input-spec.json

```diff
@@ -2052,16 +2052,10 @@
     {
         "pointer": "/solver/augmented_lagrangian/max_solver_iters",
         "min": 1,
-        "default": 500,
+        "default": 50,
         "type": "int",
         "doc": "Maximum number of iterations of the nonlinear solver inside the AL steps."
     },
-    {
-        "pointer": "/solver/augmented_lagrangian/force",
```

Comment:
Why remove this one?

Reply:
this is not used anymore

Reply:
no, it was used. but i decided to remove it? good idea?

Reply:
I don't need it, but I thought maybe Siqi was using it.